tests/src/geomodel/builder/CMakeLists.txt

```diff
@@ -31,4 +31,4 @@
 #     54518 VANDOEUVRE-LES-NANCY
 #     FRANCE
 
-add_ringmesh_test(test-build-2d-geomodels-from-3d.cpp geogram_extension geomodel_tools io)
\ No newline at end of file
+add_ringmesh_test(test-build-2d-geomodels-from-3d.cpp geomodel_tools io)
```

Comment:
Why is there this red symbol at the end?

Reply:
EOF?

Reply:
it means no new line after this one. This is the end of the file.